create_objects.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'teacher'
+
+def create_person
+  print 'Do you want to create a student (1) or a teeacher (2)? [Input the number]: '
+  number = gets.chomp.to_i
+  print 'Age: '
+  age = gets.chomp.to_i
+  print 'Name: '
+  name = gets.chomp
+  person = create_student(name, age) if number == 1
+  person = create_teacher(name, age) if number == 2
+  add_person(person)
+  puts 'Person created successfully.'
+end
```

Comment:
- [ ] Kindly fix this error 
![image](https://user-images.githubusercontent.com/24703171/210236279-6718cc04-826f-4d47-9573-aa1bce7b0261.png)
when I try to create a new student I get this error.